src/lib/sql/sql_pipeline.cpp

```diff
@@ -61,8 +62,14 @@ SQLPipeline::SQLPipeline(const std::string& sql, std::shared_ptr<TransactionCont
 
       parsed_statement->setIsValid(true);
 
-      auto pipeline_statement =
-          std::make_shared<SQLPipelineStatement>(std::move(parsed_statement), _transaction_context, use_mvcc);
+      // Get the statement string from the original query string, so we can pass it to the SQLPipelineStatement
+      DebugAssert(sql_string_offset < sql.length(),
+                  "SQL statements have a total length that is longer than the original string.");
+      const auto statement_string = boost::trim_copy(sql.substr(sql_string_offset, statement->stringLength));
+      sql_string_offset += statement->stringLength;
```

Comment:
sql_string_offset seems to be unused after this statement and is 0u before it. Remove it - or is the usage missing/overlooked by me?

Reply:
we are in a for-loop and update the offset in each iteration. in line 68 this is used to get the substr from this position onwards.